docs/pipeline-components-and-applications/snowplow-micro/index.md

```diff
@@ -0,0 +1,545 @@
+---
```

Comment:
Note: copied almost verbatim from the previous location

Reply:
"Doing things the way we have always done them" is the most dangerous phrase in the english language (closely followed by "is this grenade meant to have a pin?").

Could we use this as a chance to look at a better form for API docs, I don't know if this would work but something like  https://github.com/PaloAltoNetworks/docusaurus-openapi-docs  ?

Reply:
I like that! But would prefer not to do it in this PR :P Otherwise it’s a lot of changes at once.

Reply:
I'll raise it as an issue so we don't forget